edge/pkg/edged/kubeclientbridge/typed/core/v1/serviceaccount_bridge.go

```diff
@@ -46,6 +47,11 @@ func (c *ServiceAccountsBridge) CreateToken(ctx context.Context, serviceAccountN
 	return c.MetaClient.ServiceAccountToken().GetServiceAccountToken(c.ns, serviceAccountName, tokenRequest)
 }
 
+func (c *ServiceAccountsBridge) Delete(ctx context.Context, podUID string, opts metav1.DeleteOptions) error {
+	c.MetaClient.ServiceAccountToken().DeleteServiceAccountToken(types.UID(podUID))
+	return nil
```

Comment:
`Delete` will delete the `ServiceAccount` in the database when pod is deleted. @WillardHu PTAL
/approve